### ISSUE TITLE
Discourse - Lack of focus ring around link element on the header of the post

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#139]
**Dit probleem kan op meerdere pagina's aanwezig zijn, waaronder:**
<a href="https://community.developer.overheid.nl/t/officiele-bekendmakingen-binnenlandse-zaken-en-koninkrijksrelaties/726">Officiële bekendmakingen (Binnenlandse Zaken en Koninkrijksrelaties)</a>

**Probleembeschrijving:**
Probleem: Er is een link met de zichtbare tekst 'developer.overheid' die zich binnen de koptekst van de forumbericht-kaart bevindt. Wanneer deze link de focus krijgt, is er geen zichtbare focusring rond dit element te zien. Gebruikers die met het toetsenbord navigeren moeten kunnen zien welk element momenteel de focus heeft. Het ontbreken van focusindicatoren heeft vooral invloed op gebruikers met motorische beperkingen, visuele beperkingen en cognitieve beperkingen.

Mogelijke verbeteringen: Implementeer een duidelijk zichtbare focusindicator voor deze link (en alle andere interactieve elementen). Controleer of er CSS-code is die de focusindicator mogelijk verwijdert. Ga na of het Discourse-thema of aangepaste CSS de standaard focusstijlen overschrijft.

**Referentiecode:**
`<a href="/u/developer.overheid" data-user-card="developer.overheid" class="">developer.overheid</a>`

**Gerelateerde WCAG:**
WCAG 2.4.7 Focus visible
Level AA

**Referentieafbeelding:**

![Text developer dot overheid with no underline or focus ring ](https://github.com/user-attachments/assets/d96be370-e39b-4eeb-9a66-037896d5e257)

_**Referentienummer:** A152_
